Makefile

```diff
@@ -316,43 +297,39 @@ endif
 
 ios-framework-universal: ios-framework ios-framework-sim
 ifndef TANGRAM_IOS_FRAMEWORK_SLIM
-	@mkdir -p ${IOS_FRAMEWORK_UNIVERSAL_BUILD_DIR}/${CONFIG}
-	@cp -r ${IOS_FRAMEWORK_BUILD_DIR}/lib/${CONFIG}/TangramMap.framework ${IOS_FRAMEWORK_UNIVERSAL_BUILD_DIR}/${CONFIG}
-	lipo ${IOS_FRAMEWORK_BUILD_DIR}/lib/${CONFIG}/TangramMap.framework/TangramMap \
-		${IOS_FRAMEWORK_SIM_BUILD_DIR}/lib/${CONFIG}/TangramMap.framework/TangramMap \
-		-create -output ${IOS_FRAMEWORK_UNIVERSAL_BUILD_DIR}/${CONFIG}/TangramMap.framework/TangramMap
+	@mkdir -p ${IOS_FRAMEWORK_UNIVERSAL_BUILD_DIR}/${BUILD_TYPE}
+	@cp -r ${IOS_FRAMEWORK_BUILD_DIR}/lib/${BUILD_TYPE}/TangramMap.framework ${IOS_FRAMEWORK_UNIVERSAL_BUILD_DIR}/${BUILD_TYPE}
+	lipo ${IOS_FRAMEWORK_BUILD_DIR}/lib/${BUILD_TYPE}/TangramMap.framework/TangramMap \
+		${IOS_FRAMEWORK_SIM_BUILD_DIR}/lib/${BUILD_TYPE}/TangramMap.framework/TangramMap \
+		-create -output ${IOS_FRAMEWORK_UNIVERSAL_BUILD_DIR}/${BUILD_TYPE}/TangramMap.framework/TangramMap
 endif
 
 rpi: cmake-rpi
-	@cd ${RPI_BUILD_DIR} && \
-	${MAKE}
+	cmake --build ${RPI_BUILD_DIR}
 
 cmake-rpi:
 	@mkdir -p ${RPI_BUILD_DIR}
 	@cd ${RPI_BUILD_DIR} && \
 	cmake ../.. ${RPI_CMAKE_PARAMS}
 
 linux: cmake-linux
-	cd ${LINUX_BUILD_DIR} && \
-	${MAKE}
+	cmake --build ${LINUX_BUILD_DIR}
 
 cmake-linux:
-	mkdir -p ${LINUX_BUILD_DIR}
-	cd ${LINUX_BUILD_DIR} &&\
+	@mkdir -p ${LINUX_BUILD_DIR}
+	@cd ${LINUX_BUILD_DIR} && \
 	cmake ../.. ${LINUX_CMAKE_PARAMS}
 
 tizen-arm: cmake-tizen-arm
```

Comment:
We should these tizen out. (I said it!)

Reply:
Might make sense to remove the Tizen build, it's probably broken now anyway. Since that isn't the purpose of this PR I'll leave it as is for now.

Reply:
Yup, should be done sooner rather than later as its not officially supported and is probably broken!